src/components/biggive-search/biggive-search.scss

```diff
@@ -0,0 +1,120 @@
+.container {
+  @include standard-font();
+  @include all-round-shadow();
+  width: 400px;
+  padding: 1.5rem;
+
+  h2 {
+    @include standard-font();
+    width: 100%;
+    font-weight: 500;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    text-transform: uppercase;
+    font-size: 1.4rem;
+  }
+
+  .search-box {
+    display: flex;
+    background-color: #f1f1f1;
+    padding: 15px 10px;
+    margin-top: 1rem;
+    margin-bottom: 1.25rem;
+    border-radius: 5px;
+
+    .icon {
+      @include icon();
+      filter: opacity(0.3);
+      margin: auto;
+    }
+  
+    input {
+      width: 100%;
+      box-sizing: border-box;
+      border: none;
+      background-color: #f1f1f1;
+      border-radius: 3px;
+      font-size: 1.1rem;
+  
+      &::placeholder {
+        color: rgb(150, 150, 150);
+      }
+  
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  button {
+    @include standard-font();
+    width: 100%;
+    padding: 1.5rem 0.75rem;
+    box-sizing: border-box;
+    background-color: $colour-primary;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    text-transform: uppercase;
+    font-size: 1rem;
+
+    &:hover {
+      cursor: pointer;
+      background-color: navy;
+    }
+  }
+}
+
+@media screen and (max-width: 1024px) {
+    .container {
+      width: 350px;
+
+      h2 {
+        font-size: 1.3rem;
+      }
+
+      input {
+        font-size: 0.9rem;
+      }
+    }
+}
+
+@media screen and (max-width: 768px) {
+  .container {
+    width: 275px;
+
+    h2 {
+      font-size: 1.2rem;
+    }
+  }
+}
+
+@media screen and (max-width: 425px) {
```

Comment:
I'm a bit concerned generally that we are doing a lot of _very_ specific styling including fixed pixel values here. It's a lot of CSS + therefore quite heavy for a small component, and I'm not convinced we need it. Generally things should be responsive, fill 100% of the space they're given unless there's a compelling reason otherwise, and I think that for layout we should be avoiding fixed pixel sizes wherever possible.

Because this element will be just one piece of a larger layout, I'm not convinced media queries should be getting used for _any_ layout stuff at this point. There might be a case for varying text size but I also think this should live in a mixin so we're not hard-coding media query px breakpoints everywhere.

Reply:
@NoelLH The search component should to change its design a bit when in mobile mode according to Shape History's design of the component. See it [here](https://www.figma.com/file/YhR1S7LFBG2dx2LU9TYQgh/The-Big-Give?node-id=11%3A618), the Search Component in mobile mode looks different to what it looks like desktop mode. But on the point of mixins you are right, I will try to use them.

Reply:
@alihejazi First of all I don't think those are the latest Figma designs we've had are they? I pinned the v2 file we got to `#website-redesign` and we should update if there is anything newer. https://www.figma.com/file/prvfiPkZEqc5dLCNCjEqC0/Web-Design?node-id=388%3A2288

I also don't see how it follows that the types of variation designed should logically mean we need the component itself to have a fixed px width. Can you be more specific about why you think it need to work this way?

Reply:
I have removed the media queries, this should no longer be an issue